translations/tiled_de.ts

```diff
@@ -3899,7 +3900,7 @@ Bitte Format angeben.</translation>
     <message>
         <location line="+1"/>
         <source>Erase Terrain</source>
-        <translation type="unfinished"></translation>
+        <translation>Terrain löschen</translation>
```

Comment:
In this case, it's "erase" as in using an eraser. Are you sure "löschen" is the right word?